packages/asset-swapper/src/utils/market_operation_utils/bancor_service.ts

```diff
@@ -1,65 +1,34 @@
 import { SupportedProvider } from '@0x/dev-utils';
-import { BigNumber } from '@0x/utils';
 import { SDK } from '@bancor/sdk';
-import { Ethereum, getDecimals } from '@bancor/sdk/dist/blockchains/ethereum';
-import { fromWei, toWei } from '@bancor/sdk/dist/helpers';
-import { BlockchainType, Token } from '@bancor/sdk/dist/types';
+import { Ethereum } from '@bancor/sdk/dist/blockchains/ethereum';
+import { BlockchainType } from '@bancor/sdk/dist/types';
 
-import { BancorFillData, Quote } from './types';
+import { TOKENS } from './constants';
 
-/**
- * Converts an address to a Bancor Token type
- */
-export function token(address: string, blockchainType: BlockchainType = BlockchainType.Ethereum): Token {
-    return {
-        blockchainType,
-        blockchainId: address,
-    };
-}
+const findToken = (tokenAddress: string, graph: object): string =>
+    // If we're looking for WETH it is stored by Bancor as the 0xeee address
+    tokenAddress.toLowerCase() === TOKENS.WETH.toLowerCase()
+        ? '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
+        : Object.keys(graph).filter(k => k.toLowerCase() === tokenAddress.toLowerCase())[0];
```

Comment:
Nit: regular function

Reply:
Should we also revert if the token is not found?

Reply:
nah I want it to fail silently, like if it encounters a token that doesn't exist in Bancor. It'll just fail the lookup inside Bancor anyway